idasen/__init__.py

```diff
@@ -299,6 +300,7 @@ async def move_to_target(self, target: float):
         async def do_move():
             previous_height = await self.get_height()
             will_move_up = target > previous_height
+            last_move_time: float | None = None
```

Comment:
```suggestion
            last_move_time: Optional[float] = None
```

For older (but not EOL) python versions